src/test/scala/org/broadinstitute/dsde/rawls/expressions/JexlTest.scala

```diff
@@ -0,0 +1,72 @@
+package org.broadinstitute.dsde.rawls.expressions
```

Comment:
This file is a candidate for nuking if we end up going with Rhino. Similarly its dependencies in build.dbt.